extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -54,6 +56,27 @@ class AmpFitText extends AMP.BaseElement {
     this.textContent_ = '';
   }
 
+  /**
+   * @visibleForTesting
+   * @return {boolean}
+   */
+  isInBentoUpgradeExp() {
```

Comment:
Not sure we need to have a test for it, tbh. IMHO it's fine to inline.

Reply:
Inlined